### ISSUE TITLE
Ability to specify state_icon for Icon Element in Picture Elements

### ISSUE THREAD

**Issue by iantrich** [home-assistant/ui-schema#138]
Much like the Image Element has a state_image configuration, I believe the same would be useful for the Icon Element to have a state_icon configuration.

**Comment by balloob** [comment]
Use case?

**Comment by iantrich** [comment]
Mine is specific to battery level of my fitbit device. I get the states `Full`, `Medium` and `Low` and I wanted the ability to add styling to the icon based on that state. i.e.

`Full` -> `color: green`
`Medium` -> `color: yellow`
`Low` -> `color: red`

I was able to accomplish this using the image element and creating my own custom images, which wasn't that much work, but would not be something easily done using a frontend GUI to create such a layout. e.g. https://imgur.com/a/BhynmVj

In short, an added state_icon that supports individual styles per state opens up a lot of options for customization which is what picture element seems to be designed for.

**Comment by balloob** [comment]
state_icon != state_style though.
